frontend/src/pages/associations/list.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { PageTitle } from "../../utils/common";
+import { api } from "../../services/apiService";
+import { Card, CardBody, CardTitle } from "reactstrap";
+import { Link } from "react-router-dom";
+import { useQuery } from "react-query";
+
+export const AssociationList = () => {
+    const { data: associations, isLoading, error } = useQuery(
+        "associations.list",
+        api.associations.list
+    );
+
+    if (isLoading) return "Loading...";
+    if (error) {
+        return `Something went wrong: ${error.message}`;
+    }
+    if (associations) {
+        return (
+            <>
+                <PageTitle>Associations</PageTitle>
+                <div className={"row"}>
```

Comment:
There is a component `Row`.
Also, [Bootstrap](https://getbootstrap.com/docs/4.3/layout/grid/) always puts it in a `Container` component (I think this is not the case here)

Reply:
`Row` is in react-bootstrap at list, I don't know about reactstrap